crates/sui-framework/sources/package.move

```diff
@@ -201,7 +201,7 @@ module sui::package {
         UpgradeTicket {
             cap: object::id(cap),
             package,
-            policy: cap.policy,
+            policy,
```

Comment:
Thanks for fixing this!  Can you add a regression test to `crates/sui-frameworks/tests/package_tests.move` for this? You might need to expose some accessors for `UpgradeTicket`.